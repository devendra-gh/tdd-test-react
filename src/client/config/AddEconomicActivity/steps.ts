export interface IStep {
  name: string;
  subSteps?: string[];
}

export const addEconomicActivitySteps: IStep[] = [
  { name: 'addEconomicActivity.addActivity' },
  { name: 'addEconomicActivity.getDEDApproval' },
];
