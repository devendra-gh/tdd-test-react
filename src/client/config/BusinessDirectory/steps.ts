export interface IStep {
  name: string;
  subSteps?: string[];
}
