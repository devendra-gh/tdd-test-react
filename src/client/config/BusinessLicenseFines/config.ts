import { getVariables } from './services/bpm';
import businessLicenseFinesPages from './pages';

interface IState {}

const initialState: IState = {
  businessKey: '',
  instanceId: '',
  formBusinessLicenceFine: {
    licenceNo: '',
    isLoading: false,
  },
  stepsStatus: {},
};

const redirectUrl =
  window.location.href.indexOf('stage.tamm.abudhabi') !== -1
    ? 'https://stage.tamm.abudhabi/'
    : 'https://www.tamm.abudhabi/';

const config = {
  appName: 'businessServiceApplication',
  // configuration/app version, please keep updateding
  // for now redux counts on this to disregard the old state
  version: '6',
  // default variables, for now title is used
  defaults: {
    title: 'default.title',
  },
  getVariables,
  // initial redux state, you can configure per your project need,
  initialState,
  header: {
    template: 'header',
    props: {
      aspectsOfLifeType: 'business-management',
      breadcrumbs: [
        {
          label: 'breadcrumb.home',
          link: `${redirectUrl}`,
          linkTarget: '_self',
        },
        {
          label: 'breadcrumb.digitalServices',
          link: `${redirectUrl}tamm-centers-services`,
          linkTarget: '_self',
        },
        {
          label: 'breadcrumb.dedServices',
          link: `${redirectUrl}tamm-centers-services/department-of-economic-development`,
          linkTarget: '_self',
        },
      ],
    },
    state: {
      mapState: ['user', 'locale', 'title'],
      mapDispatch: ['user', 'locale'],
    },
  },
  footer: {
    template: 'footer',
    state: {
      mapState: ['user'],
    },
  },
  pages: [...businessLicenseFinesPages],
};

export default config;
