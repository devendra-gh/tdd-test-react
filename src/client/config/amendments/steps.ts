export interface IStep {
  name: string;
  subSteps?: string[];
  id?: string;
}

export const steps: IStep[] = [
  {
    name: 'process.selectLicenceNumber',
  },
  {
    name: 'process.selectAmendmentType',
  },
  {
    id: 'amendments',
    name: 'process.makeAmendment',
  },
  {
    name: 'process.getDEDApproval',
  },
  {
    name: 'process.makePayment',
  },
  {
    name: 'process.downloadLicence',
  },
];

export default steps;
