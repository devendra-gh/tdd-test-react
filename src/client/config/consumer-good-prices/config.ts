import bpm from 'client/services/bpm';
import { IVariables } from '@tamm/app-composer';
import pages from './pages';

interface IState {}

export const initialState: IVariables = {
  itemId: null,
  subTitle: '',
  currentGoods: [],
  goodsList: {
    selectedGood: '',
    loadedCategoriesLocale: 'en',
    nonPaginatedData: [],
    data: [],
    categories: [],
    alertText: '',
    alertStatus: '',
    searchBy: 'product',
    selectedCategory: '',
    searchText: '',
    currentPage: 1,
    totalCount: null,
    recInPage: 10,
    batchDate: null,
    showTable: false,
    showSpinner: false,
    showAlert: false,
    showCategories: false,
  },
  showSpinner: false,
  retailers: [
    { value: '1', label: 'KM TRADING', checked: false },
    { value: '2', label: 'LULU', checked: false },
    { value: '3', label: 'SPAR', checked: false },
  ],
  locations: [
    { value: '1', label: 'Al Ain', checked: false },
    { value: '2', label: 'Abu Dhabi', checked: false },
    { value: '3', label: 'Sharjah', checked: false },
  ],
};

const redirectUrl =
  window.location.href.indexOf('stage.tamm.abudhabi') !== -1
    ? 'https://stage.tamm.abudhabi/'
    : 'https://www.tamm.abudhabi/';

const config = {
  appName: 'consumerGoodPrices',
  // configuration/app version, please keep updateding
  // for now redux counts on this to disregard the old state
  version: '1',
  // default variables, for now title is used
  defaults: {
    title: 'Consumer Good Prices',
  },
  getVariables: bpm.getVariables,
  // initial redux state, you can configure per your project need,
  initialState,
  header: {
    template: 'header',
    props: {
      aspectsOfLifeType: 'business-management',
      breadcrumbs: [
        {
          label: 'breadcrumb.home',
          link: `${redirectUrl}`,
          linkTarget: '_self',
        },
        {
          label: 'breadcrumb.digitalServices',
          link: `${redirectUrl}tamm-centers-services`,
          linkTarget: '_self',
        },
        {
          label: 'breadcrumb.dedServices',
          link: `${redirectUrl}tamm-centers-services/department-of-economic-development`,
          linkTarget: '_self',
        },
      ],
    },
    state: {
      mapState: ['user', 'locale', 'title', 'form', 'minimumSteps'],
      mapDispatch: ['user', 'locale'],
    },
    // hideForPaths: ['/login'],
  },
  footer: {
    template: 'footer',
    state: {
      mapState: ['user'],
    },
    // hideForPaths: ['/login'],
  },
  // pages of the application
  pages,
};

export default config;
