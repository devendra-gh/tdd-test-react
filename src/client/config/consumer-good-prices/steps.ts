// list of substeps ids
export interface IStep {
  name: string;
  subSteps?: string[];
}

export const minimumSteps: IStep[] = [];
const steps: IStep[] = [];

export default steps;
