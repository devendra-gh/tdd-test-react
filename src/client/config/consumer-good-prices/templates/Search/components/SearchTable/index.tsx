import React from 'react';
import * as PropTypes from 'prop-types';
import { withTemplateHooks, IVariables } from '@tamm/app-composer';
import RadioGroup from '@tamm/ui-lib-v2-radio-group';
import Grid from '@tamm/ui-lib-v2-grid';
import Table from '@tamm/ui-lib-v2-table';
import Pagination from '@tamm/ui-lib-v2-pagination';
import moment from 'moment';
import { spacer } from '../../../../utils';

export const noRefCheck = () => '';

const getRadioForTable = (id: string, list: any[], props: IVariables) => {
  let label = '';
  list.find((elem: any) => {
    if (elem.id === id) {
      label = elem.enName;
      return true;
    }
    return false;
  });
  return (
    <RadioGroup.Radio
      checked={props.goodsList.selectedGood === id}
      disabled={false}
      label={label}
      name={id}
      value={id}
      onChange={() => {
        props.handleGoodsRadioChange(id, props);
      }}
    />
  );
};

getRadioForTable.propTypes = {
  goodsList: PropTypes.shape({
    selectedGood: PropTypes.string.isRequired,
  }).isRequired,
  handleGoodsRadioChange: PropTypes.func.isRequired,
};

function SearchTable(props: IVariables) {
  const { Col, Row } = Grid;
  const { goodsList, i18n } = props;
  return (
    <>
      {goodsList.showTable && goodsList && !!goodsList.data.length && (
        <React.Fragment>
          <Row>
            <Col span={6}>
              <p>
                {i18n('global.displayingSearch', {
                  numberOfRecords: goodsList.data.length,
                  totalRecords: goodsList.totalCount,
                })}
              </p>
            </Col>
            <Col span={6} style={{ textAlign: 'right' }}>
              <p>
                {i18n('cgp_prices_uploading_date')}
                {`: ${moment(goodsList.batchDate).format('ll')}`}
              </p>
            </Col>
          </Row>
          {spacer('15px')}
          <Table
            i18n={i18n}
            clickable
            columns={[
              {
                id: 'checkbox',
                title: i18n('global.englishName'),
                render: (_content: string, recordId: string) => {
                  return getRadioForTable(recordId, goodsList.data, props);
                },
              },
              {
                align: 'end',
                id: 'arName',
                title: i18n('global.arabicName'),
              },
            ]}
            headerHidden={false}
            items={goodsList.data}
            size="default"
            title={i18n('cgp_list_of_goods')}
          />
          {spacer('40px')}
          {goodsList.totalCount / goodsList.recInPage > 1 && (
            <>
              <Pagination
                current={props.goodsList.currentPage}
                disabled={false}
                i18n={noRefCheck}
                pageSize={props.goodsList.recInPage}
                total={props.goodsList.totalCount}
                onChange={(currentPage: number) =>
                  props.handlePageChange(currentPage, props)
                }
              />
            </>
          )}
        </React.Fragment>
      )}
    </>
  );
}

export default withTemplateHooks(SearchTable);
