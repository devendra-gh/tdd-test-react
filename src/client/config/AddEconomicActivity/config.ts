import '@tamm/ui-lib-v2-styles/common.less';
import '@tamm/ui-lib-v2-styles/colors.less';
import './override.less';
import { IVariables } from '@tamm/app-composer';
import { addEconomicActivitySteps } from './steps';
import pages from './pages';
import { getVariables } from './services/bpm';

export const initialState: IVariables = {
  fileUploadData: {
    documents: {
      activitySupportingDoc: null,
    },
  },

  instanceId: '',
  businessKey: '',

  formData: {
    arabicActivityName: '',
    englishActivityName: '',
    arabicActivityDescription: '',
    englishActivityDescription: '',
    name: '',
    email: '',
    mobileNumber: '',
  },
  helperData: {
    isSubmitted: false,
  },
  newActivityApiData: {
    submittedOn: '',
    altId: '',
  },
  statusRecieved: false,
  stepsStatus: {},
  addEconomicActivitySteps,
};

const redirectUrl =
  window.location.href.indexOf('stage.tamm.abudhabi') !== -1
    ? 'https://stage.tamm.abudhabi/'
    : 'https://www.tamm.abudhabi/';

const config = {
  appName: 'newActivityService',
  // configuration/app version, please keep updateding
  // for now redux counts on this to disregard the old state
  version: '0.6',
  // default variables, for now title is used
  defaults: {
    title: 'default.title',
  },
  getVariables,
  // initial redux state, you can configure per your project need,
  initialState,
  header: {
    template: 'header',
    props: {
      aspectsOfLifeType: 'manage_business',
      breadcrumbs: [
        {
          label: 'breadcrumb.home',
          link: `${redirectUrl}`,
          linkTarget: '_self',
        },
        {
          label: 'breadcrumb.digitalServices',
          link: `${redirectUrl}tamm-centers-services`,
          linkTarget: '_self',
        },
        {
          label: 'breadcrumb.dedServices',
          link: `${redirectUrl}tamm-centers-services/department-of-economic-development`,
          linkTarget: '_self',
        },
      ],
    },
    state: {
      mapState: ['user', 'locale', 'title'],
      mapDispatch: ['user', 'locale'],
    },
  },
  footer: {
    template: 'footer',
    state: {
      mapState: ['user'],
    },
  },
  // pages of the application
  pages,
};

export default config;
