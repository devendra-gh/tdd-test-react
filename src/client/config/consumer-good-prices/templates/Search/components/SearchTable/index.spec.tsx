import React from 'react';
import { render, cleanup, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import { MemoryRouter } from 'react-router';
import { Viewport, ViewportProvider } from '@tamm/ui-lib-v2-viewport';
import { steps } from 'client/config/steps';
import SearchTable, { noRefCheck } from './index';

jest.mock('@tamm/app-composer', () => ({
  withTemplateHooks: (component: any) => component,
}));

// eslint-disable-next-line no-console
console.log = () => {};
console.info = () => {};
console.warn = () => {};
console.error = () => {};

describe('consumer-good-prices/templates/Search', () => {
  let props: any;

  beforeEach(() => {
    Object.defineProperty(window, 'matchMedia', {
      writable: true,
      value: jest.fn().mockImplementation(query => ({
        matches: false,
        media: query,
        onchange: null,
        addListener: jest.fn(), // deprecated
        removeListener: jest.fn(), // deprecated
        addEventListener: jest.fn(),
        removeEventListener: jest.fn(),
        dispatchEvent: jest.fn(),
      })),
    });

    props = {
      handleGoodsRadioChange: jest.fn(),
      handlePageChange: jest.fn(),
      noRefCheck: jest.fn(),
      changePage: jest.fn(),
      onChange: jest.fn(),
      onCancel: jest.fn(),
      title: 'title',
      subTitle: 'subTitle',
      text: 'text',
      history: {
        push: jest.fn(),
      },
      showTable: true,
      actions: {
        title: {
          update: jest.fn(),
        },
        goodsList: {
          update: jest.fn(),
        },
        showSpinner: {
          update: jest.fn(),
        },
        currentChecked: {
          update: jest.fn(),
        },
      },
      locale: 'en',
      i18n: jest.fn(i => i),
      currentStep: '',
      currentSubStep: '',
      steps,
      stepsStatus: {},
      goodsList: {
        batchDate: '2020-02-09T06:23:37.803Z',
        selectedGood: 'selectedGood',
        showTable: true,
        currentPage: 1,
        totalCount: 100,
        recInPage: 10,
        data: [
          {
            id: '123',
            enName: 'goods',
            value: 1,
            arName: 'goods',
          },
          {
            id: null,
            enName: 'goods',
            value: 1,
            arName: 'goods',
          },
        ],
      },
      disabled: false,
      displayErrorFlag: true,
      showSpinner: true,
      info: true,
      error: true,
      description: 'sample description',
      isCategorySelected: true,
      searchVal: 'abcd',
      currentChecked: '',
      buttons: [
        {
          id: 1,
          label: 'button.next1',
          onClick: jest.fn(),
          uiType: 'primary',
          withArrow: true,
          alignIcon: 'end',
        },
      ],
      searchBy: 'product',
      radioGroups: [
        {
          id: 'product',
          label: 'cgp_by_product',
        },
        {
          id: 'category',
          label: 'cgp_by_cat',
        },
      ],
      itemsFound: [
        {
          Id: '7D2F3F99-2844-4711-9660-5E3DAE82FC1B',
          ItemNameEn: 'Sugar Cubes / SIS / 450 g / Australia',
          ItemNameAr: 'سكر مكعبات اس اي اس / 450 جرام/ أستراليا',
          Barcode: '8888167217989',
          StoreItemJob: {
            Id: '2020-01-04-1578171601705',
            StartDate: '2020-01-04T21:00:01.705Z',
            EndDate: null,
            Active: true,
            Status: 1,
            CollectionDate: '2020-01-03',
          },
        },
      ],
      categories: [{ id: 'Bread and Cereals', label: 'Bread and Cereals' }],
    };
  });

  afterEach(cleanup);

  test('renders with props', () => {
    const { container } = render(
      <MemoryRouter>
        <ViewportProvider>
          <Viewport sm md lg xl>
            <SearchTable {...props} />
          </Viewport>
        </ViewportProvider>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  test('clicking actions should call corret functions', () => {
    const { container } = render(
      <MemoryRouter>
        <ViewportProvider>
          <Viewport sm md lg xl>
            <SearchTable {...props} />
          </Viewport>
        </ViewportProvider>
      </MemoryRouter>,
    );
    noRefCheck();
    // fireEvent.click(getByLabelText('Prev'));
    fireEvent.click(container.querySelectorAll('.ui-lib-radio__input')[0]);
    expect(props.handleGoodsRadioChange).toBeCalled();
    expect(props.handlePageChange).not.toBeCalled();
  });
});
