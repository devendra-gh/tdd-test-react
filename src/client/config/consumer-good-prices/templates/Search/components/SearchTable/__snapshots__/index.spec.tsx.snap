// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`consumer-good-prices/templates/Search renders with props 1`] = `
<div>
  <div
    class="ui-lib-viewport ui-lib-viewport_lg"
  >
    <div
      class="ant-row ui-lib-grid-row ui-lib-grid-row "
      style="margin-left: -10px; margin-right: -10px;"
    >
      <div
        class="ant-col ui-lib-grid-col ui-lib-grid-col-12 ui-lib-grid-col "
        style="padding-left: 10px; padding-right: 10px;"
      >
        <p>
          global.displayingSearch
        </p>
      </div>
      <div
        class="ant-col ui-lib-grid-col ui-lib-grid-col-12 ui-lib-grid-col "
        style="padding-left: 10px; padding-right: 10px; text-align: right;"
      >
        <p>
          cgp_prices_uploading_date
          : Feb 9, 2020
        </p>
      </div>
    </div>
    <div
      style="height: 15px;"
    />
    <div
      class="ui-lib-table"
    >
      <div
        class="ui-lib-table__pre"
      >
        <div
          class="ui-lib-table__pre-title ui-lib-heading_h4"
        >
          cgp_list_of_goods
        </div>
      </div>
      <div
        class="ui-lib-table__root"
        data-round-bottom="true"
        data-round-top="false"
      >
        <table
          class="ui-lib-table__root-grid"
        >
          <thead
            class="ui-lib-table-header"
          >
            <tr
              class="ui-lib-table-row"
              data-highlighted="false"
              data-ui-type="default"
              data-underlined="false"
              style="vertical-align: top;"
            >
              <td
                aria-label="checkbox"
                class="ui-lib-table-cell"
                data-align="start"
                data-collapsed="false"
                data-disabled="false"
                data-fit-content-width="false"
                data-hoverable="false"
                data-is-extra="false"
                data-is-header="true"
                data-size="default"
                data-sortable="false"
                data-ui-type="header"
                role="gridcell"
                tabindex="0"
              >
                <span
                  class="ui-lib-table-cell_bdi"
                >
                  global.englishName
                </span>
                
              </td>
              <td
                aria-label="arName"
                class="ui-lib-table-cell"
                data-align="end"
                data-collapsed="false"
                data-disabled="false"
                data-fit-content-width="false"
                data-hoverable="false"
                data-is-extra="false"
                data-is-header="true"
                data-size="default"
                data-sortable="false"
                data-ui-type="header"
                role="gridcell"
                tabindex="0"
              >
                <span
                  class="ui-lib-table-cell_bdi"
                >
                  global.arabicName
                </span>
                
              </td>
            </tr>
          </thead>
          <tbody>
            <tr
              class="ui-lib-table-row"
              data-highlighted="false"
              data-ui-type="regular"
              data-underlined="true"
              style="vertical-align: top;"
            >
              <td
                aria-label="checkbox"
                class="ui-lib-table-cell"
                data-align="start"
                data-collapsed="false"
                data-disabled="false"
                data-fit-content-width="false"
                data-hoverable="false"
                data-is-extra="false"
                data-is-header="false"
                data-size="default"
                data-sortable="false"
                data-ui-type="regular"
                role="gridcell"
                tabindex="0"
              >
                <label
                  class="ui-lib-radio ui-lib-radio_default"
                  for="uil_radio_button.1"
                >
                  <input
                    class="ui-lib-radio__input"
                    data-index="0"
                    id="uil_radio_button.1"
                    name="123"
                    type="radio"
                    value="123"
                  />
                  <div
                    class="ui-lib-radio__box"
                  >
                    <div
                      class="ui-lib-radio__box-dot"
                    />
                  </div>
                  <div
                    class="ui-lib-radio-label"
                  >
                    goods
                  </div>
                </label>
                
              </td>
              <td
                aria-label="arName"
                class="ui-lib-table-cell"
                data-align="end"
                data-collapsed="false"
                data-disabled="false"
                data-fit-content-width="false"
                data-hoverable="false"
                data-is-extra="false"
                data-is-header="false"
                data-size="default"
                data-sortable="false"
                data-ui-type="regular"
                role="gridcell"
                tabindex="0"
              >
                <span
                  class="ui-lib-table-cell_bdi"
                >
                  goods
                </span>
                
              </td>
            </tr>
            <tr
              class="ui-lib-table-row"
              data-highlighted="false"
              data-ui-type="regular"
              data-underlined="false"
              style="vertical-align: top;"
            >
              <td
                aria-label="checkbox"
                class="ui-lib-table-cell"
                data-align="start"
                data-collapsed="false"
                data-disabled="false"
                data-fit-content-width="false"
                data-hoverable="false"
                data-is-extra="false"
                data-is-header="true"
                data-size="default"
                data-sortable="false"
                data-ui-type="header"
                role="gridcell"
                tabindex="0"
              >
                <label
                  class="ui-lib-radio ui-lib-radio_default"
                  for="uil_radio_button.2"
                >
                  <input
                    class="ui-lib-radio__input"
                    data-index="0"
                    id="uil_radio_button.2"
                    type="radio"
                    value=""
                  />
                  <div
                    class="ui-lib-radio__box"
                  >
                    <div
                      class="ui-lib-radio__box-dot"
                    />
                  </div>
                  <div
                    class="ui-lib-radio-label"
                  >
                    goods
                  </div>
                </label>
                
              </td>
              <td
                aria-label="arName"
                class="ui-lib-table-cell"
                data-align="end"
                data-collapsed="false"
                data-disabled="false"
                data-fit-content-width="false"
                data-hoverable="false"
                data-is-extra="false"
                data-is-header="true"
                data-size="default"
                data-sortable="false"
                data-ui-type="header"
                role="gridcell"
                tabindex="0"
              >
                <span
                  class="ui-lib-table-cell_bdi"
                >
                  goods
                </span>
                
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      
    </div>
    <div
      style="height: 40px;"
    />
    <div
      class="ui-lib-pagination"
    >
      <div
        aria-label=""
        class="ui-lib-pagination__item ui-lib-pagination__item_page ui-lib-pagination__item_current"
        role="button"
        tabindex="0"
      >
        1
      </div>
      <div
        aria-label=""
        class="ui-lib-pagination__item ui-lib-pagination__item_page"
        role="button"
        tabindex="0"
      >
        2
      </div>
      <div
        aria-label=""
        class="ui-lib-pagination__item ui-lib-pagination__item_page"
        role="button"
        tabindex="0"
      >
        3
      </div>
      <div
        aria-label=""
        class="ui-lib-pagination__item ui-lib-pagination__item_page"
        role="button"
        tabindex="0"
      >
        4
      </div>
      <div
        aria-label=""
        class="ui-lib-pagination__item ui-lib-pagination__item_page"
        role="button"
        tabindex="0"
      >
        5
      </div>
      <div
        aria-label="Next 3 Pages"
        class="ui-lib-pagination__item ui-lib-pagination__item_dots"
        role="button"
        tabindex="0"
      >
        <svg
          class="ui-lib-pagination__item-icon ui-lib-icon"
          data-key=""
          height="24"
          tabindex="0"
          version="1.1"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title />
          <circle
            cx="3.25"
            cy="12"
            r="3.25"
          />
          <circle
            cx="12"
            cy="12"
            r="3.25"
          />
          <circle
            cx="20.75"
            cy="12"
            r="3.25"
          />
        </svg>
      </div>
      <div
        aria-label=""
        class="ui-lib-pagination__item ui-lib-pagination__item_page"
        role="button"
        tabindex="0"
      >
        10
      </div>
      <div
        aria-label="Next"
        class="ui-lib-pagination__item ui-lib-pagination__item_nav"
        role="button"
        tabindex="0"
      >
        <svg
          class="ui-lib-pagination__item-icon ui-lib-icon"
          data-key=""
          height="24"
          tabindex="0"
          version="1.1"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title />
          <path
            d="M23.987,12a2.411,2.411,0,0,0-.814-1.8L11.994.361a1.44,1.44,0,0,0-1.9,2.162l8.637,7.6a.25.25,0,0,1-.165.437H1.452a1.44,1.44,0,0,0,0,2.88H18.563a.251.251,0,0,1,.165.438l-8.637,7.6a1.44,1.44,0,1,0,1.9,2.161L23.172,13.8A2.409,2.409,0,0,0,23.987,12Z"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
